jmix-dynattr/dynattr/src/main/java/io/jmix/dynattr/model/JavaFieldNameSupportedSymbols.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2024 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.dynattr.model;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+
+import java.lang.annotation.*;
+
+@Target(ElementType.FIELD)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@Constraint(validatedBy = JavaFieldNameSupportedSymbolsValidator.class)
+public @interface JavaFieldNameSupportedSymbols {
+    String message() default "The field can only contain latin letters or digits. First letter have to be a latin letter.";
```

Comment:
Current default message is more strict than logic of the validator. Use the same text from the EN messages.properties.

Reply:
Fixed